serverless/admin_guide/serverless-admin-metrics.adoc

```diff
@@ -6,7 +6,13 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Metrics enable cluster administrators to monitor how {ServerlessProductName} cluster components and workloads are performing.
+ifdef::openshift-enterprise[]
+Metrics enable cluster administrators to monitor how {ServerlessProductName} cluster components and workloads are performing. You can view different metrics for {ServerlessProductName} by navigating to xref:../../monitoring/reviewing-monitoring-dashboards.adoc#reviewing-monitoring-dashboards-admin_reviewing-monitoring-dashboards[*Dashboards*] in the {product-title} web console *Administrator* perspective.
```

Comment:
I was trying to make sure they both appeared as 2 sentences in a paragraph so that it would fit the Jupiter abstract requirements, but updated it now.